great_expectations/expectations/row_conditions.py

```diff
@@ -34,11 +34,7 @@ def _set_notnull(s, l, t) -> None:  # noqa: E741 # ambiguous name `l`
 
 
 WHITESPACE_CHARS = " \t"
-column_name = Combine(
-    Suppress(Literal('col("'))
```

Comment:
Do you understand why removing `Suppress` allows this to continue working? I don't see us referencing `column_name.column` anywhere? Why not leave `Suppress` in there? I'm not very familiar with `pyparsing`.

Reply:
so when you use `setResultsName` on a portion of a combined matcher it only uses the portion that you care calling that method on.

So the full expression basically _matches_ for `col("<anytext>")` but will only set `<anytext>` under the key `column` in the results dict.